Bad luck apparently:
```
18:19:47 start: start-smb-windows.sh
18:19:47 name: smb-windows
18:20:48 ....................
18:20:48 [ERROR] Server not reachable
18:20:48 stop: stop-smb-windows.sh
```

--- a/core/Command/Integrity/CheckApp.php
+++ b/core/Command/Integrity/CheckApp.php
@@ -65,6 +65,9 @@ class CheckApp extends Base {
 		$path = strval($input->getOption('path'));
 		$result = $this->checker->verifyAppSignature($appid, $path);
 		$this->writeArrayInOutputFormat($input, $output, $result);
+		if (count($result)>0){
+			return 1;
+		}
 	}
 
 }

--- a/core/Command/Integrity/CheckCore.php
+++ b/core/Command/Integrity/CheckCore.php
@@ -59,5 +59,8 @@ class CheckCore extends Base {
 	protected function execute(InputInterface $input, OutputInterface $output) {
 		$result = $this->checker->verifyCoreSignature();
 		$this->writeArrayInOutputFormat($input, $output, $result);
+		if (count($result)>0){
+			return 1;
+		}
 	}
 }

